tests/tools_tests/src/expected/DocExtractionRes.res.json

```diff
@@ -57,18 +69,39 @@
       "filepath": "src/DocExtractionRes.res",
       "line": 23,
       "col": 5
+    },
+    "detail": 
+    {
+      "kind": "signature",
+      "details": {
+      "parameters": [{
+          "path": "t"
+        }],
+      "returnType": {
+        "path": "t"
+      }
+    }
     }
   }, 
   {
-    "id": "DocExtractionRes.SomeConstant",
+    "id": "DocExtractionRes.\\\"SomeConstant\"",
```

Comment:
Not sure why this changed now? Is there an `Ext_ident.unwrap_uppercase_exotic` missing somewhere?

Reply:
Thanks, that was an oversight